src/pages/standard-report/HtmlReport.js

```diff
@@ -81,7 +84,7 @@ class HtmlReport extends Component {
                 width="100%"
                 height={this.state.height}
                 seamless={true}
-                sandbox="allow-same-origin allow-scripts allow-modals"
+                sandbox="allow-same-origin allow-scripts allow-modals allow-downloads"
```

Comment:
What's this one for?

Reply:
See #136 - some templates would include custom download buttons and in the latest chrome it wouldn't be allowed to use them. This attribute helps with that.
(It's unrelated to the current problem, but I snuck it in anyway)